Moderation-Policy.md

```diff
@@ -0,0 +1,48 @@
+## Moderation policy
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js Github Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
+* The Moderation Policy is openly documented as part of the Working Group or Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions taken.
+
+### Terms
+
+* "Collaborator" refers to any individual member or "outside collaborator" of the Node.js Github Organization.
+* "Post" refers to the content of any issue, pull request or comment.
+* "Moderate" refers to the act of modifying the content of or deleting any Post that is in obvious violation of the Node.js Code of Conduct.
+* "Ban" refers to the act of blocking an individual Github account from any further participation in the Node.js Github Organization.
+
+### Policy
+
+* All Posts are expected to respect the Node.js Code of Conduct.
+* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
+* Collaborator's must not Moderate any Post authored by another Collaborator without first giving the author an opportunity of at least 24 hours to modify or remove the Post on their own. In extreme circumstances the TSC can be consulted to waive the 24 hour grace period.
```

Comment:
* Moderate is defined further up in the document. Posting a new comment that calls out any code of conduct violation is separate
* Extreme = obvious gross violations of the code of conduct
* The 8chan/4chan situation involved non-Collaborator posts, which are addressed separately from this bullet point.
* The 24hrs starts from the request to edit/change. I can clarify this.

The motivation is transparency and collaboration while giving fellow collaborators the benefit of the doubt. Note that moderation of non-Collaborators is handled differently.